lib/vagrant-sshfs/cap/linux/sshfs_mount.rb

```diff
@@ -164,8 +163,28 @@ def self.sshfs_slave_mount(machine, opts, hostpath, expanded_guest_path)
           #          stdin <= r2        pipe2         w2 <= stdout 
           #
           # Wire up things appropriately and start up the processes
-          p1 = spawn(sftp_server_cmd, :out => w2, :in => r1, :err => t1)
-          p2 = spawn(ssh_cmd,         :out => w1, :in => r2, :err => t2)
+          if Vagrant::Util::Platform.windows?
+            # Need to handle Windows differently. Kernel.spawn fails to work, if the shell creating the process is closed.
+            # See https://github.com/dustymabe/vagrant-sshfs/issues/31
+            Process.create(:command_line => sftp_server_cmd,
+                           :creation_flags => Process::DETACHED_PROCESS,
+                           :process_inherit => false,
+                           :thread_inherit => true,
+                           :startup_info => {:stdin => w2, :stdout => r1, :stderr => t1})
+
+            Process.create(:command_line => ssh_cmd,
+                           :creation_flags => Process::DETACHED_PROCESS,
+                           :process_inherit => false,
+                           :thread_inherit => true,
+                           :startup_info => {:stdin => w1, :stdout => r2, :stderr => t2})
+          else
+            p1 = spawn(sftp_server_cmd, :out => w2, :in => r1, :err => t1, :pgroup => true)
+            p2 = spawn(ssh_cmd,         :out => w1, :in => r2, :err => t2, :pgroup => true)
```

Comment:
I
Not sure about the new :pgroup => true . I'm tempted the leave it the way 
it was because that seems to be working pretty well.


I think using the pgroup flag is important here. You want to make sure that 
the spawned process gets its own process group. Otherwise the process might 
receive signals it is not supposed to get or you get zombie processes.